Tests/CryptoTests/Digests/DigestsTests.swift

```diff
@@ -67,9 +67,9 @@ class DigestsTests: XCTestCase {
     }
     
     func testMD5() {
-        XCTAssert(Data(Insecure.MD5.hash(data: Data())).count == Insecure.MD5.byteCount)
-        try! XCTAssert(Data(Insecure.MD5.hash(data: Data())) == Data(hexString: "d41d8cd98f00b204e9800998ecf8427e"))
-        try! XCTAssert(Data(Insecure.MD5.hash(data: Data("abcdbcdecdefdefgefghfghighijhijkijkljklmklmnlmnomnopnopq".utf8))) == Data(hexString: "8215ef0796a20bcaaae116d3876c664a"))
+        XCTAssertEqual(Data(Insecure.MD5.hash(data: Data())).count, Insecure.MD5.byteCount)
+        try! XCTAssertEqual(Data(Insecure.MD5.hash(data: Data())), Data(hexString: "d41d8cd98f00b204e9800998ecf8427e"))
```

Comment:
I think we should change the signature if this function to be throwing, and remove the `!` from `try!`. As seen in many of the `AESGCMTests` tests.